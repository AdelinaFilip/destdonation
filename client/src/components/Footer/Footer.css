.main-footer {
  color: #e1ffcb;
  background: #658c47;
  padding-top: 0em;
  bottom: 0;
  width: 100%;
  position: relative;
}

.rows {
  color: #e1ffcb;
  align-items: center;
  text-align: center;
}
