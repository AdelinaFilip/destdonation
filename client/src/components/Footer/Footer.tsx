import React from "react";
import "./Footer.css";
import Facebook from "./facebook.svg";
import Insta from "./instagram.svg";
import Twitter from "./twitter.svg";

const Footer = () => {
  return (
    <div className="main-footer">
      <div className="container">
        <div className="rows">
          <a
            href={"https://www.cityharvest.org.uk/"}
            target="_blank"
            rel="noopener noreferrer"
            style={{ color: "#E1FFCB" }}
          >
            www.cityharvest.org.uk &nbsp;
          </a>
          <a href={"https://www.facebook.com/CityHarvestLondon/"}>
            <img src={Facebook} width={20} alt={""}></img>
          </a>{" "}
          &nbsp;
          <a href={"https://www.instagram.com/cityharvestlondon/?hl=en"}>
            <img src={Insta} width={20} alt={""}></img>
          </a>{" "}
          &nbsp;
          <a href={"https://twitter.com/cityharvest_ldn?lang=en"}>
            <img src={Twitter} width={20} alt={""}></img>
          </a>{" "}
          &nbsp; &copy;
          {new Date().getFullYear()} DestDonation
        </div>
      </div>
    </div>
  );
};
export default Footer;
