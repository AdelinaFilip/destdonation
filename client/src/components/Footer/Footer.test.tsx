import React from "react";
import { shallow } from "enzyme";

import Footer from "./Footer";

describe("Footer", () => {
  it("should render correctly and match snapshot", () => {
    const component = shallow(<Footer />);

    expect(component).toMatchSnapshot();
  });
});
