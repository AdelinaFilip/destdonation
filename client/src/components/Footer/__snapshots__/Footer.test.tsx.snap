// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly and match snapshot 1`] = `ShallowWrapper {}`;
